pkg/detectors/jiratoken/v1/jiratoken_test.go

```diff
@@ -16,7 +16,7 @@ var (
 	invalidTokenPattern  = "Z7VoI?J0K4rF#LBfkhO&LAWX"
 	validDomainPattern   = "hereisavalidsubdomain.heresalongdomain.com"
 	invalidDomainPattern = "?y4r3fs1ewqec12v1e3tl.5Hcsrcehic89saXd.ro@"
-	validEmailPattern    = "xfKF_BZq7@grum.com"
```

Comment:
@kashifkhan0771 Not sure but the old input is also valid email and should pass the user case.